### DISCUSSION TITLE
2023/04/18/%E6%AF%8F%E6%97%A5%E4%B8%80fun/

### DISCUSSION THREAD

**Discussion by giscus** [FloranceYeh/FloranceYeh.github.io#5] [Announcements]
# 2023/04/18/%E6%AF%8F%E6%97%A5%E4%B8%80fun/

It's Florance!

https://floranceyeh.netlify.app/2023/04/18/%E6%AF%8F%E6%97%A5%E4%B8%80fun/

**Comment by FloranceYeh** [comment]
$6^{666}$

**Comment by ERA-YES** [comment]
拜托...
